### PR TITLE
fix: EVA on Ray bugs

### PR DESCRIPTION
**Pull request by kaushikravichandran**
This PR fixes some bugs in the path of the optimizer when using EVA on Ray

### CODE REVIEW

File: eva/optimizer/optimizer_tasks.py [hunk 1]
```diff
@@ -199,6 +199,7 @@ def execute(self):
                     self.optimizer_context.task_stack.push(
                         ExploreGroup(group, self.optimizer_context)
                     )
+        print("out of the loop")
```
Comment (gaurav274) [inline eva/optimizer/optimizer_tasks.py:202]: Remove this

File: eva/optimizer/optimizer_tasks.py [hunk 2]
```diff
@@ -281,7 +282,10 @@ def execute(self):
         for child_id in self.root_expr.children:
             child_grp = memo.get_group_by_id(child_id)
             if child_grp.get_best_expr(PropertyType.DEFAULT):
-                cost += child_grp.get_best_expr_cost(PropertyType.DEFAULT)
+                # Note: May never get hit when using EVA on Ray
```
Comment (gaurav274) [inline eva/optimizer/optimizer_tasks.py:285]: Any reason for this comment?

**Review by gaurav274 — COMMENTED** [review]
Remove `yolov5.pt`

File: eva/optimizer/optimizer_tasks.py (hunk 2 above)
Comment (kaushikravichandran) [reply eva/optimizer/optimizer_tasks.py:285]: Jsut for future ref. Will remove it soon

File: eva/optimizer/optimizer_tasks.py (hunk 1 above)
Comment (kaushikravichandran) [reply eva/optimizer/optimizer_tasks.py]: Done

**Review by gaurav274 — APPROVED** [review]

**Pull request merged**